test/performance/scale_from_zero_test.go

```diff
@@ -183,5 +179,6 @@ func TestScaleFromZero5(t *testing.T) {
 }
 
 func TestScaleFromZero50(t *testing.T) {
+	t.Skip()
```

Comment:
We should include an issue number that tracks re-enabling this test.

Reply:
Done.